packages/example-log-extension/README.md

```diff
@@ -35,10 +35,10 @@ import {
 class LogApp extends LogLevelMixin(RestApplication) {
   constructor() {
     super({
-      components: [LogComponent],
       logLevel: LOG_LEVEL.ERROR,
```

Comment:
This `logLevel` configuration may need update as part of https://github.com/strongloop/loopback-next/pull/952 or after it's landed. I am not sure, I'd just like to bring this to our attention - I guess mostly yours, @raymondfeng .